### DISCUSSION TITLE
Where can I find weewx-wdc-interceptor-docker on docker-hub?

### DISCUSSION THREAD

**Discussion by johnfux** [Daveiano/weewx-wdc#133] [Q&A]
Hey @Daveiano,

I am using weewx for the last several year and I want to migrate my installation vom the Raspberry Pi into a Docker-Container. Looking araound the internet for a nice docker container with interceptor support, I ran into your GitHub project.

I want to install the docker container on a Synology DiskStation, but the software has the docker-hub-repository as default.

Have you released your project (https://github.com/Daveiano/weewx-wdc-interceptor-docker) as a Docker-Image on  Docker-Hub?
I searched a lot, but couldn't find it. Probably there isn't a release on that site, is it?

Thanks for a short reply. 

Regards Johannes

**Comment by Daveiano** [comment]
Hi Johannes,

no, I never pushed it to docker hub, sorry.

But you can build it yourself with a few commands:

```
git pull https://github.com/Daveiano/weewx-wdc-interceptor-docker.git && cd weewx-wdc-interceptor-docker
docker build . -t "weewx"
```

Hope that helps!

**Reply by johnfux** [reply]
Thank you for the quit reply. I will have a look and try to figure it out... :-)
